api/api-feeds-api-v1/src/main/java/com/thoughtworks/go/apiv1/feedsapi/FeedsApiControllerV1.java

```diff
@@ -76,6 +79,8 @@ public void setupRoutes() {
             get(Routes.FeedsAPI.JOB_XML, this.mimeType, this::jobXML);
             get(Routes.FeedsAPI.SCHEDULED_JOB_XML, this.mimeType, this::scheduledJobs);
             get(Routes.FeedsAPI.MATERIAL_URL, this.mimeType, this::materialXML);
+
+            after("/*", this::setContentType);
```

Comment:
`this.mimeType`?? may be `after("/*", this.mimeType, this::setContentType);`

Reply:
- **Failure:** send error message based on the requested accept header type. This will allow the client to interpret the error message as per the requested types.

- **Success:** Feed APIs always sends a response as XML. Hence expected content types can be one from `application/xml`, `text/xml`, `application/rss+xml` or `application/atom+xml`. Content-type will be set based on the Accept header. Defaults to `application/xml` if accept header contains non XML content type

Reply:
> this.mimeType?? may be after("/*", this.mimeType, this::setContentType);

Not sure if that will make any difference